src/events/eventActions.js

```diff
@@ -178,6 +179,12 @@ export const registerAndStartPolling =
             globalSettings: getGlobalSettings(),
           },
         );
+        logging.error(e, {
+          version: e.version.raw(),
```

Comment:
Let's… hmm, I was going to say let's include something like `coarseVersion: e.version.coarse()` here, but I think we can actually do better — before this `logging.error` call, let's call:
```js
    logging.setTagsFromServerVersion(e.version);
```

That way we'll get the information as Sentry "tags", which are more flexibly usable than "extras"; and we can drop this `version` field in the extras.